### ISSUE TITLE
Clarify doc for sound

### ISSUE THREAD

**Issue by fred-wang** [rpgjs/rpgjs-documentation#1]
See 

https://docs.rpgjs.dev/classes/map.html#sounds
https://docs.rpgjs.dev/classes/sound.html#properties

How would you specify a list of sounds sound1, sound2, sound3, ... soundN that play in loop? It seems the current API only allows one sound to be played in a loop but I can't verify because of https://github.com/RSamaium/RPG-JS/issues/87

**Comment by RSamaium** [comment]
Indeed, you can put several sounds on a card in order to play music at the same time

I added some elements in the doc to better understand and I have added a guide (which can be completed) on how to use a sound:

https://docs.rpgjs.dev/guide/create-sound.html
